src/main/java/Storage.java

```diff
@@ -0,0 +1,132 @@
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Storage {
+
+    public static ArrayList<Task> data;
+    private static final Path fileLocation = Paths.get("./data", "duke.txt");
+    private static final Path directoryLocation = Paths.get("./data");
+
+    private static void createFile() {
+        try {
+            Files.createFile(fileLocation);
+        } catch (Exception e) {
+            System.out.println(e);
+        }
+    }
+
+    private static boolean checkForFile() {
+        boolean fileExists = Files.exists(fileLocation);
+        return fileExists;
+    }
+
+    private static boolean checkForDirectory() {
```

Comment:
Perhaps the method name could be changed to isDirectoryAvailable() to make is sound more like a boolean method.